pyqir-generator/src/python.rs

```diff
@@ -30,11 +33,16 @@ impl PyQIR {
         }
     }
 
+    #[allow(clippy::needless_pass_by_value)]
+    #[allow(clippy::unnecessary_wraps)]
+    #[allow(clippy::unused_self)]
     fn add_measurement(&mut self, qubit: String, target: String) -> PyResult<()> {
         log::info!("measure {} => {}", qubit, target);
         Ok(())
     }
 
+    #[allow(clippy::needless_pass_by_value)]
+    #[allow(clippy::unnecessary_wraps)]
```

Comment:
Is it necessary to return `PyResult<()>` instead of `()`?

Reply:
When I looked originally, I thought so, we can check again.

Reply:
@samarsha  We can remove the `PyResult<()>` for anything which doesn't return errors. I'll start on that.

Reply:
@samarsha Pushed changes.